src/utils/getHexFromBytes.ts

```diff
@@ -11,7 +11,7 @@ import {hexFormatOptions, IHexFormatOptions} from '../config.js';
  * input: [2, 5, 12, 255, 105, 139, 125]
  * output: '02 05 0c ff 69 8b 7d' or '0x02 0x05 0x0c 0xff 0x69 0x8b 0x7d'
  */
-export default ( buffer: Uint8Array, options: IHexFormatOptions = {} ): string => {
+export default ( buffer: Uint8Array | Array<number>, options: IHexFormatOptions = {} ): string => {
```

Comment:
the idea seems valid but the result may be unpredictable as now you can pass `[1, 2, 333333]` in this function

Reply:
fixed. use Uint8Array everywhere